src/main/java/eod/card/concrete/normal/SingleContract.java

```diff
@@ -21,8 +22,9 @@ public SingleContract() {
     @Override
     public void applyEffect(EffectExecutor executor) {
         WarObject[]  characters = WarObject(player.getBoard()).which(OwnedBy(player.rival())).which(Being(Character.class)).which(WithoutStatus(Status.SNEAK)).get();
+        ((Character) player.selectObject(characters)).damage(new DamageParam(4));
         executor.tryToExecute(
-                Damage(4, Effect.HandlerType.Rival).onOneOf(player, characters)
+                Damage(new DamageParam(4), Effect.HandlerType.Rival).onOneOf(player, characters)
```

Comment:
This will damage two selected characters.
(Contradicted with Card's description)

Reply:
fixed